src/main/java/gov/cms/madie/models/measure/TestCaseGroupPopulation.java

```diff
@@ -14,7 +14,7 @@
 @Builder(toBuilder = true)
 @NoArgsConstructor
 @AllArgsConstructor
-@ValidScoringPopulation
+//@ValidScoringPopulation
```

Comment:
should this be commented out?

Reply:
Yes, We are removing these validations as a quick way to handle QDM-related changes, There will be a tech debt story to enhance these API validations in the future.